src/tests/ftest/pool/list_verbose.py

```diff
@@ -0,0 +1,210 @@
+            errors.append(
+                "NVMe free too small! Expected at least {}; Actual {}".format(
+                    expected_nvme_free_min, actual_nvme_free))
+
+        # Disable target 7 in rank 1.
+        status = self.pool.exclude(ranks=[1], tgt_idx="7")
+        if status:
+            # Verify disabled is 1.
+            output = self.pool.dmg.get_pool_list_all(verbose=True)
+            targets_disabled = output[0]["targets_disabled"]
+            targets_total = output[0]["targets_total"]
+            if targets_disabled != 1:
+                errors.append(
+                    "Unexpected targets_disabled! {}".format(targets_disabled))
+
+            # Also verify total is still 8.
+            expected_target = self.params.get(
+                "targets", "/run/server_config/servers/*")
+            if targets_total != expected_target:
+                errors.append(
+                    "Unexpected targets_total! {}".format(targets_total))
+        else:
+            errors.append("dmg pool exclude failed!")
+
+        if errors:
+            self.fail("\n----- Errors detected! -----\n{}".format(
+                "\n".join(errors)))
+
+        self.destroy_containers(containers=self.container)
+        self.destroy_pools(pools=self.pool)
+
+    def verify_used_imbalance(self, storage):
+        """Verification steps for test_used_imbalance.
+
+        Args:
+            storage (str): NVME or SCM.
+
+        Returns:
+            list: Errors.
+
+        """
+        # 1. Create a pool with 80GB.
+        if storage == "NVME":
+            self.add_pool(namespace="/run/pool_both/*")
+        else:
+            self.add_pool(namespace="/run/pool_scm_only/*")
+
+        # 2. Store free.
+        output = self.pool.dmg.get_pool_list_all(verbose=True)
+        free_before = -1
+        for usage in output[0]["usage"]:
+            if usage["tier_name"] == storage:
+                free_before = usage["free"]
+                break
+
+        # 3, 4. Create an S1 container and run 1GB IOR with S1.
+        self.run_ior_with_pool()
+
+        # 5, 6. Verify the free and imbalance.
+        output = self.pool.dmg.get_pool_list_all(verbose=True)
+        free_after = 0
+        imbalance = 0
+        for usage in output[0]["usage"]:
+            if usage["tier_name"] == storage:
+                free_after = usage["free"]
+                imbalance = usage["imbalance"]
+                break
+
+        errors = []
+        ior_write_size = self.params.get("ior_write_size", "/run/*")
+        if free_after > free_before - ior_write_size:
+            errors.append(
+                "{} free is bigger than expected! Before {}; After {}".format(
+                    storage, free_before, free_after))
+
+        exp_imbalance = self.params.get(storage.lower(), "/run/exp_imbalance/*")
+        # Imbalance can be 10 or 11.
+        if imbalance <= exp_imbalance - 2 or imbalance >= exp_imbalance + 3:
```

Comment:
Would this be simpler?  The allowable value list could even come from the test yaml.
```suggestion
        if imbalance not in [10,11]:
```